book/security.md

```diff
@@ -792,9 +792,9 @@ def request(self, payload=None):
         params = {}
         if ";" in cookie:
             cookie, rest = cookie.split(";", 1)
-            for param_pair in rest.split(";"):
+        for param_pair in rest.split(";"):
```

Comment:
Confused by this change. Either we need to initialize `rest` as the empty string, or the `for` loop needs to be inside the `if` statement. But maybe this code block was already misshapen? Or there's a spaces-vs-tabs issue?

Reply:
This change was not on purpose, and seems to be a whitespace issue. Fixed.